examples/testing-module/moduleframework-cmd

```diff
@@ -1 +1 @@
-../../tools/bashhelper.py
\ No newline at end of file
+../../moduleframework/bashhelper.py
```

Comment:
I guess bashhelper.py scipt alone shown not be executed right?
If it is executed, then please add it to setup.py, especially console_scripts.

Reply:
I have to move it, to make it working again. it caused that tools direcotory in in /usr/share/moduleframework/tools and not in /usr/share/python*/site-packages/moduleframework, thats why it is not accessible for python